src/psyclone/tests/psyir/transformations/transformations_test.py

```diff
@@ -274,6 +275,36 @@ def test_omptargettrans(sample_psyir):
     assert loops[0].parent.parent is loops[1].parent.parent
 
 
+def test_ompdeclaretargettrans(sample_psyir, fortran_writer):
```

Comment:
Since we're here and it's just the one, please could you change line 214 to use an f-string.

Reply:
Done